src/modules/Schizophrenia/index.tsx

```diff
@@ -0,0 +1,65 @@
+import CleanSound from './soundclean_schizo.mp3';
+import DiseaseSound from './sounds_schizo.mp3';
+import { HeadsetIcon, OnIcon, OffIcon } from './customIcons';
+import { Button } from 'antd';
+import { PauseCircleOutlined, PlayCircleOutlined  } from '@ant-design/icons';
+import { useRef, useState } from 'react';
+
+import './style.css';
```

Comment:
could need a little clean up :

```suggestion
import { useRef, useState } from 'react';
import { Button } from 'antd';
import { PauseCircleOutlined, PlayCircleOutlined  } from '@ant-design/icons';

import './style.css';
import { HeadsetIcon, OnIcon, OffIcon } from './customIcons';
import CleanSound from './soundclean_schizo.mp3';
import DiseaseSound from './sounds_schizo.mp3';
```